lms/envs/aws.py

```diff
@@ -1101,3 +1101,10 @@
 ########################## Derive Any Derived Settings  #######################
 
 derive_settings(__name__)
+
+#RACCOONGANG
+if AUTH_TOKENS.get('RG_SENTRY_DSN', None):
+    import sentry_sdk
+    from sentry_sdk.integrations.django import DjangoIntegration
+    sentry_sdk.init(AUTH_TOKENS.get('RG_SENTRY_DSN'), integrations=[DjangoIntegration()])
```

Comment:
I would also add `CeleryIntegration()` into the integrations to the `lms` app.

Reply:
please assign task to developer

Reply:
I've updated PR with the Celery integration configuration.

Reply:
please provide link to ticket with results of celery task monitoring testing